src/common/wrap/wrap-events.js

```diff
@@ -2,34 +2,38 @@
  * Copyright 2020 New Relic Corporation. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-
+/**
+ * This module is used directly by: session_trace
+ * It is also called by -> wrapXhr <-, so see "wrap-xhr.js" for featuers that uses this indirectly.
+ */
 import {ee as baseEE} from '../event-emitter/contextual-ee'
-import { createWrapperWithEmitter as wfn } from './wrap-function'
+import { createWrapperWithEmitter as wfn, unwrapFunction } from './wrap-function'
 import { getOrSet } from '../util/get-or-set'
 import { globalScope, isBrowserScope } from '../util/global-scope'
 
 const wrapped = {}
+var XHR = XMLHttpRequest
+var ADD_EVENT_LISTENER = 'addEventListener'
+var REMOVE_EVENT_LISTENER = 'removeEventListener'
```

Comment:
Use `const` or `let`

Reply:
Based on the capitalization of the variable names, I assume these are meant to be constants. That would make `const` the better choice. If they are getting re-assigned, then the name should be camelCased and `let` should be used.